src/main/java/com/example/onlineshop/model/Basket.java

```diff
@@ -0,0 +1,61 @@
+package com.example.onlineshop.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.GenerationType;
+import javax.persistence.Column;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.ManyToMany;
+import javax.persistence.FetchType;
+import javax.persistence.CascadeType;
+import javax.persistence.JoinTable;
+
+@Setter
+@Getter
+@EqualsAndHashCode
+@ToString
+
+@Entity
+@Table(name = "baskets", schema = "spring_shop")
+public class Basket {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", unique = true, nullable = false)
```

Comment:
do we need to use unique = true, nullable = false here? if we use annotation id this parameters is on by default